tools/cockpit.spec

```diff
@@ -156,6 +153,13 @@ The Cockpit Web Service listens on the network, and authenticates users.
 %prep
 %setup -q
 
+# Apply patches using git
```

Comment:
Sure, it's turing complete. But I was trying to keep it simple here and handle both cases the same, give broader testing etc.